### ISSUE TITLE
Latest version missing "open in app" link when opened on android

### ISSUE THREAD

**Issue by braintweak** [jitsi/jitsi-meet#13373]
### Description:
Latest jitsi self hosted release (2.0.8615) for some reason is missing "open in app" link, that was present in previous versions.
Previous versions, in my case, 2.0.8319 and 2.0.8044, have the link present when opening the room URL on mobile.

### Steps to reproduce:

1. Open link, leading to conference room in current version of jitsi selfhosted (https://meeting.foobar.com/test).
2. See blank page with "video chat is not available on mobile" screen with no option to open in app, unless its hosted on supported URLs, baked into the app.

### Expected behavior:

Previously there were links for opening in browser and in app, and it worked just fine (see pic2). Self hosted URLs opened in app.

### Actual behavior:

No links to open in app are present on current version.

### Server information:

- Jitsi Meet version: 2.0.8615
- Operating System: Ubuntu 20.04 or 22.04

### Client information:

- Browser / app version: Firefox mobile / Chrome mobile
- Operating System: Android 12 or Android 13

Pic1:
![pic1](https://github.com/jitsi/jitsi-meet/assets/9693830/1d3a022d-3eb7-4c39-b750-56aa399d77c0)
Pic2:
![pic2](https://github.com/jitsi/jitsi-meet/assets/9693830/439d4409-49fa-4cd7-9df4-1fa4d2fe2e57)

**Comment by saghul** [comment]
Something might have gone wrong with your update.

That screen got a new UI:

![image](https://github.com/jitsi/jitsi-meet/assets/317464/b232a4b7-a13e-4924-baf5-026457b3b1bf)

You seem to be seeing the old one.

Do you have any CSS changes you did manually?

**Comment by braintweak** [comment]
> Something might have gone wrong with your update.

Hmm... Haven't considered that. I'll have a look after a fresh clean install. Its a straight install via apt following the install guide (I mean no docker/etc).
I thought its reproducible since it happens on 2 different systems I've set up. 

> Do you have any CSS changes you did manually?

The only things I've touched were background pictures and logo by replacing the file/modifying config, fixing some translation files, but did not touch CSS.

Thank you.
